### PR TITLE
Feature/logs roles home

### PR DESCRIPTION
**Pull request by varun257**
### Requirements

https://interne.ovh.net/projects/browse/UML-9
Feature to list roles, add, edit and delete roles
View role summary

### Description of the Change
- Roles home page wher list of roles are listed
- Add role using modal window
- Edit role using modal window
- Delete a role
- View overview/summary of the role

### Benefits
Helps user to manage roles

### Possible Drawbacks

None

### Applicable Issues
None

### CODE REVIEW

File: client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_description"></dt>
+            <dd class="preview-list__definition"
+                data-ng-bind=":: ctrl.role.info.description"></dd>
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_permissions"></dt>
+            <dd class="preview-list__definition">
+                <span data-ng-if="ctrl.role.permissions.length === 0"> - </span>
+                <div data-ng-if="ctrl.role.permissions.length > 0">
+                    <div data-ng-repeat="permission in ctrl.role.permissions">
```
Comment (Netnix) [inline client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html]: You should always have a "track by" with a ng-repeat, even if it's $index.

File: client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                        <ul class="pl-0" data-ng-if="permission.alias">{{permission.alias.name}} - {{'logs_role_overview_alias' | translate}}</ul>
+                        <ul class="pl-0" data-ng-if="permission.dashboard">{{permission.dashboard.title}} - {{'logs_role_overview_dashboard' | translate}}</ul>                        
+                    </div>
+                </div>
+            </dd>
+            <dt class="preview-list__term"
+                data-translate="logs_role_overview_members"></dt>
+            <dd class="preview-list__definition">
+                <span data-ng-if="ctrl.role.members.length === 0"> - </span>
+                <ul class="pl-0" data-ng-repeat="member in ctrl.role.members" data-ng-bind="member.username"></ul>
```
Comment (Netnix) [inline client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html]: You should always have a "track by" with a ng-repeat, even if it's $index.

File: client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html (hunk 1 above)
Comment (varun257) [reply client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html]: @Netnix  the logic changed a bit. Have a look now. Have added track by index too

File: client/app/dbaas/logs/detail/roles/add/logs-role-add.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<div class="oui-modal oui-modal_shadow">
```
Comment (Mboulianne) [inline client/app/dbaas/logs/detail/roles/add/logs-role-add.html]: You should use the components in modal.component.js

Comment (varun257) [reply client/app/dbaas/logs/detail/roles/add/logs-role-add.html]: I tried using the component, but it still renders cui-modal, which means i will have to overwrite some properties on oui-modal__body like background-color manually. Is it okay ?

Or is there anyway to render oui-modal component directly using ControlHelper ?

Comment (varun257) [reply client/app/dbaas/logs/detail/roles/add/logs-role-add.html]: And also if i do change it will be a hack until the oui modal works properly

Comment (Mboulianne) [reply client/app/dbaas/logs/detail/roles/add/logs-role-add.html]: Ah I didn't know the modal yo uwere using is not the same visually as cui-modal.  Sorry

**Pull request merged**